tnc/static.py

```diff
@@ -161,6 +148,9 @@ class FRAME_TYPE(Enum):
     ARQ_DC_OPEN_ACK_W = 226
     ARQ_DC_OPEN_N = 227
     ARQ_DC_OPEN_ACK_N = 228
+    ARQ_MANUAL_01 = 230
+    # ...
```

Comment:
Same thought here with removing like above 🤔

Reply:
The latest push removes the manual mode code.